en/contributing/test.md

```diff
@@ -1,90 +1,85 @@
 # Testing
 
-The SDK has both unit and integration tests, written using the [Google C++ Test Framework](https://github.com/google/googletest/blob/master/googletest/docs/Primer.md) (`gtest`).
+The SDK has both unit and integration tests, written using the [Google C++ Test Framework](https://github.com/google/googletest/blob/master/googletest/docs/Primer.md) \(`gtest`\).  
 The unit tests are run every time new code is committed to the SDK codelines, and must pass before the code can be merged.
 
 This topic shows how to run the existing tests.
 
-> **Tip** For information on *writing* tests see: [Writing Plugins > Test Code](../contributing/plugins.md#testing).
-
+> **Tip** For information on _writing_ tests see: [Writing Plugins &gt; Test Code](../contributing/plugins.md#testing).
 
 ## Running Unit Tests
 
 To run all unit tests:
 
 ```
-make run_unit_tests
+./build/default/src/unit_tests_runner
 ```
 
-
 ## Running Integration Tests
 
-Tests can be run against the simulator (either manually starting PX4 SITL or letting the tests start it automatically) or against a real vehicle.
+Tests can be run against the simulator \(either manually starting PX4 SITL or letting the tests start it automatically\) or against a real vehicle.
 
-> **Tip** To run SITL you will need to install the *Gazebo* simulator.
-This is included as part of the standard PX4 installation for [macOS](https://dev.px4.io/en/setup/dev_env_mac.html)
-and [Linux](https://dev.px4.io/en/setup/dev_env_linux.html#development-toolchain). It does not run on Windows.
+> **Tip** To run SITL you will need to install the _Gazebo_ simulator.  
+> This is included as part of the standard PX4 installation for [macOS](https://dev.px4.io/en/setup/dev_env_mac.html)  
+> and [Linux](https://dev.px4.io/en/setup/dev_env_linux.html#development-toolchain). It does not run on Windows.
 
 ### Autostart PX4 SITL
 
 Make sure that the [PX4 Gazebo simulation](https://dev.px4.io/en/simulation/gazebo.html) is built and works:
 
 ```
-cd wherever/Firmware/
+cd path/to/Firmware/
 make px4_sitl gazebo
 ```
 
 Then press **Ctrl+C** to stop the simulation and run the integration tests:
 
 ```
-cd wherever/DronecodeSDK/
-AUTOSTART_SITL=1 make run_integration_tests
+cd path/to/DronecodeSDK/
+AUTOSTART_SITL=1 ./build/debug/src/integration_tests/integration_tests_runner
 ```
 
-To run the tests without the 3D viewer (*gzclient*), use:
+To run the tests without the 3D viewer \(_gzclient_\), use:
 
 ```
-AUTOSTART_SITL=1 HEADLESS=1 make run_integration_tests
+AUTOSTART_SITL=1 HEADLESS=1 ./build/debug/src/integration_tests/integration_tests_runner
 ```
 
 ### Run PX4 SITL Manually
 
 Build and run the PX4 simulation manually:
 
 ```
-cd wherever/Firmware/
+cd path/to/Firmware/
 make px4_sitl gazebo
 ```
 
 Then run the tests as shown:
-```
-make run_integration_tests
-```
-
-### Run With a Real Vehicle
```

Comment:
Is this deletion of running on a real vehicle section intentional?

If so, might be worth just adding the warning back at the end of the section:

> **Warning** Some of the tests might not be suited for real vehicles, especially the takeoff and kill test!

Reply:
To me this was adding noise. If somebody has absolutely no clue to the point where they may not realize that killing a real vehicle is dangerous, then probably we don't want to give them instructions on how to do that. And if they understand the difference between a simulator and a real vehicle, they probably have no issues running their code on both, do they?

Also we don't run them on real vehicles, so I would not consider that as "supported". I don't want to have to care about only adding "safe" integration tests just because somebody may blindly follow this documentation :sweat_smile:. What if somebody has a filter to not run the "dangerous" tests, and then we add a new kill test that doesn't get filtered and we crash the drone?

That's an opinion, I can bring it back if you want, that's your call!

Reply:
A lot of people are idiots, but then most of those don't bother to read the instructions. I would be tempted to add just a note at the end. It can omit specific tests. Something like:

> **Warning** The tests are designed to run in simulation, and may not be safe if run on a real vehicle.

Reply:
I added the warning :+1: